sdk/eventhub/azure-eventhub/azure/eventhub/_pyamqp/aio/_transport_async.py

```diff
@@ -149,6 +149,36 @@ async def send_frame(self, channel, frame, **kwargs):
         await self.write(data)
         # _LOGGER.info("OCH%d -> %r", channel, frame)
 
+class AsyncTransport(
```

Comment:
can we copy changes to SB pyamqp as well?

Reply:
yes these changes will have to go back there too

Reply:
sorry, I meant can you copy them over in this PR? 🙂
or do we want to do that later?

Reply:
lol my reply wasnt clear, Ill do it in this PR itself :)